mod/profiles.php

```diff
@@ -518,23 +509,27 @@ function profiles_post(App $a) {
 function profile_activity($changed, $value) {
 	$a = get_app();
 
-	if(! local_user() || ! is_array($changed) || ! count($changed))
+	if (! local_user() || ! is_array($changed) || ! count($changed)) {
 		return;
+	}
 
-	if($a->user['hidewall'] || get_config('system','block_public'))
+	if ($a->user['hidewall'] || get_config('system','block_public')) {
 		return;
+	}
 
-	if(! get_pconfig(local_user(),'system','post_profilechange'))
+	if (! get_pconfig(local_user(),'system','post_profilechange')) {
 		return;
+	}
 
 	require_once('include/items.php');
```

Comment:
There is tons of these still in code. I think I bring them down with future PRs/branches.